jni/msp_displayport_mux.c

```diff
@@ -74,6 +74,8 @@ static uint8_t serial_passthrough = 1;
 static uint8_t compress = 0;
 static uint8_t no_btfl_hd = 0;
 
+uint16_t fc_vtx_channel = -1;
```

Comment:
an uint (unsigned int) can never be nagative. either use 0 as the indicator for not set or change it to an int so you can use negative values.

this might work because the -1 will still translate to some value, but this pattern can lead to unexpected behavior.

also, since we're doing c and being precise this should probably be a int8 rather than 16, we don't need more than 256 values.

Reply:
You are totally right, I started it as simply value to store the value and the lack of practice with C did the rest. I will move it to int8, it will always store small values as we don't have more than 8 channels.